src/bundle/Resources/public/js/alloyeditor/src/widgets/ez-embed-base.js

```diff
@@ -239,7 +239,7 @@ const embedBaseDefinition = {
     createEmbedPreview: function(title) {
         return `
             <svg class="ez-icon ez-icon--medium ez-icon--secondary">
-                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#tag"></use>
+                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/bundles/ezplatformadminui/img/ez-icons.svg#embed"></use>
```

Comment:
I wonder, shouldn't we also include built of `ezPluginEmbed` into the PR, because `./src/plugins/ez-embed.js` imports `./src/widgets/ez-embed-base` and we do not build `./src/widgets/ez-embed-base`?  ping @dew326

Reply:
Yes, the `ezPluginEmbed.js` and `ezPluginEmbed.js.map` should be added.